modules/reporting/app/models/cost_query/operator.rb

```diff
@@ -71,4 +71,30 @@ def modify(query, field, *values)
       query
     end
   end
+
+  new "=_child_work_packages", validate: :integers, label: :label_is_work_package_with_descendants do
+    def modify(query, field, *values)
+      wp_ids = []
+      values.each do |value|
+        wp_ids += ([value] << WorkPackage.find(value).descendants.map(&:id))
+      end
+      "=".to_operator.modify query, field, wp_ids
+    rescue ActiveRecord::RecordNotFound
+      query
+    end
+  end
+
+  new "!_child_work_packages", validate: :integers, label: :label_is_not_work_package_with_descendants do
+    def modify(query, field, *values)
+      wp_ids = []
+      values.each do |value|
+        value.to_s.split(",").each do |id|
+          wp_ids += ([id] << WorkPackage.find(id).descendants.map(&:id))
```

Comment:
I guess you'd want to restrict it to visible descendants here. Also, why not descendants.visible.pluck(:id) ?

Reply:
This is a 1:1 copy of the operator above ... Did not bother optimizing... Will do :)